lib/components/src/Accordion/AccordionContext.tsx

```diff
@@ -0,0 +1,19 @@
+import { createContext } from 'react';
+
+export type AddToMapFn = (id: string) => void;
+
+export type OpenMap = Record<string, boolean>;
+
+interface AccordionContextProps {
+  openState: OpenMap;
+  addToMap: AddToMapFn;
+  onOpen: (id: string) => void;
+  onClose: (id: string) => void;
+  lined: boolean | undefined;
+  bordered: boolean | undefined;
+  indentBody: boolean | undefined;
+  narrow: boolean | undefined;
+  preventToggle: boolean | undefined;
+}
```

Comment:
I think we should move these interfaces to it's own file, and de-duplicate them all. So there's 1 single source of truth for all these types, and they can't go out of sync over time.

Reply:
> I think we should move these interfaces to it's own file, and de-duplicate them all. So there's 1 single source of truth for all these types, and they can't go out of sync over time.

I like that idea